Linked list Trickiest Questions/Detecting and Removing loop from Linked List.cpp

```diff
@@ -0,0 +1,99 @@
+#include<iostream>
+#include<conio.h>
+using namespace std;
+
+struct node
+{
+  int info;

```

Comment:
Use 4 space indentation

Reply:
@SakshiShreya Mam may you please explain the way to indent the code using 4 spaces. And do I have to make this change to the entire code or to only these four lines that you just mentioned?

Reply:
@saumyashrm2802 you have to add space indentation in entire code. You can convert tab indentation to space indentation right from the text editor. If you're using Sublime/VSCode, do the following:
- ctrl+shift+P
- Search for "indentation"
- Select "Convert tabs to spaces" or "Indentation to Spaces"
- Save the file and commit

Reply:
@marwahmanbir Thankyou so much for the clarification. I have corrected the indentation.